models/gpg_key_test.go

```diff
@@ -103,6 +103,7 @@ MkM/fdpyc2hY7Dl/+qFmN5MG5yGmMpQcX+RNNR222ibNC1D3wg==
 =i9b7
 -----END PGP PUBLIC KEY BLOCK-----`
 	keys, err := checkArmoredGPGKeyString(testGPGArmor)
+	assert.NotEmpty(t, keys)
```

Comment:
So in reality the line below this is going to do this test for us anyway - there will be a panic if keys is empty.

Now the way to avoid the panic would be to do:

```
if len(keys) == 0 {
  assert.NotEmpty(t, keys)
  return
}
``` 

or even to just drop to `t.Log("checkArmoredGPGkeyString has return empty keys")` instead of using `assrt.NotEmpty` when we know it's empty.

Reply:
The asserts return booleans for these cases.